store/iavl/store.go

```diff
@@ -273,7 +273,13 @@ func (st *Store) Export(version int64) (*iavl.Exporter, error) {
 	if !ok || tree == nil {
 		return nil, fmt.Errorf("iavl export failed: unable to fetch tree for version %v", version)
 	}
-	return tree.Export(), nil
+	exporter := tree.Export()
+
+	// cosmos/iavl immutable_tree.go and export.go FAIL if tree.ndb is nil
```

Comment:
Files cannot fail, so what exactly is this comment trying to convey? Perhaps replace it with the actual methods/functions that would fail instead?